extensions/amp-social-share/amp-social-share.md

```diff
@@ -204,6 +206,20 @@ You can use [global AMP variables substitution](https://github.com/ampproject/am
 </amp-social-share>
 ```
 
+### Migrating from 0.1
+
+The `1.0` version of `amp-social-share` has a few additional features that differentiate it from the `0.1` version.
+
+1. The default icons for pre-configured providers have been updated to reflect the
+   most up-to-date logos and colors from their associated companies so expect a new
+   set of logos when migrating to `1.0`.
+2. Both the foreground and background colors of the default icons can now be styled
```

Comment:
@nainar can you advise on if a disclaimer should be here for brand guideline violations?

Reply:
Yes please let's have a disclaimer:
"Please note that when customizing brand logos ensure that you are meeting brand guidelines."

Reply:
Should we also include this here?  It's included in the Custom Styles section here: https://github.com/ampproject/amphtml/pull/31920/files#diff-c87481dd1ffac22579d5f6e496937d3385d3e92236af2737d49f4349cc07e4dcR270 (line 270)